tox.ini

```diff
@@ -18,13 +18,13 @@ commands = python -W error::DeprecationWarning -W error::PendingDeprecationWarni
 commands =
     python -m coverage erase
     python -m coverage run --source {envsitepackagesdir}/websockets,tests -m unittest {posargs}
-    python -m coverage report --show-missing --fail-under=100
+    python -m coverage report --show-missing --omit src/websockets/legacy/async_timeout/__init__.py  --fail-under=100
```

Comment:
I made changes in 399db68d to configure this in setup.cfg rather than in all coverage commands. It should be possible to remove this change.